apps/web/src/pages/settings/organization/OrganizationName.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Input, successMessage } from '@novu/design-system';
+import { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useRenameOrganization } from '../../../api/hooks';
+import { css } from '../../../styled-system/css';
+import { Stack } from '../../../styled-system/jsx';
+
+type FormValues = {
+  name: string;
+};
+
+const INPUT_HEIGHT = '50px';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏‏If you rebase your branch off of mine (`ia-webhook`), this value is now available within Panda as a semantic token!